lib/experimental/register-webfonts-from-theme-json.php

```diff
@@ -6,12 +6,11 @@
  */
 
 /**
- * Register webfonts defined in theme.json.
+ * Register webfonts defined in theme.json
+ *
+ * @param array $settings The theme.json file.
  */
-function gutenberg_register_webfonts_from_theme_json() {
-	// Get settings.
-	$settings = WP_Theme_JSON_Resolver_Gutenberg::get_merged_data()->get_settings();
-
+function _gutenberg_register_webfonts_from_theme_json( $settings ) {
```

Comment:
Should we prefix this function with `_wp` instead of `_gutenberg`? This will end up in WP Core at some point.

Reply:
Makes sense to me 👍